mmengine/runner/runner.py

```diff
@@ -1975,16 +1975,22 @@ def resume(self,
             if (previous_gpu_ids is not None and len(previous_gpu_ids) > 0
                     and len(previous_gpu_ids) != self._world_size):
                 # TODO, should we modify the iteration?
-                self.logger.info(
-                    'Number of GPU used for current experiment is not '
-                    'consistent with resuming from checkpoint')
                 if (self.auto_scale_lr is None
-                        or not self.auto_scale_lr.get('enable', False)):
+                        or self.auto_scale_lr.get('enable', False)):
                     raise RuntimeError(
-                        'Cannot automatically rescale lr in resuming. Please '
-                        'make sure the number of GPU is consistent with the '
-                        'previous training state resuming from the checkpoint '
-                        'or set `enable` in `auto_scale_lr to False.')
+                        'Number of GPU used for current experiment is not '
+                        'consistent with resuming from checkpoint, '
+                        'which may result in poor performance due to the '
+                        'learning rate. If you want the learning rate to '
+                        'be automatically adjusted, set the '
+                        '`auto_scale_lr` parameter for Runner and make '
+                        '`auto_scale_lr["enable"]=True`.')
```

Comment:
This suggestion incorporates some small aesthetic tweaks but importantly it changes the language to be more imperative (please consider my comment above).

```suggestion
                        'Number of GPUs used for current experiment is not '
                        'consistent with the checkpoint being resumed from. '
                        'This will result in poor performance due to the '
                        'learning rate. You must set the '
                        '`auto_scale_lr` parameter for Runner and make '
                        '`auto_scale_lr["enable"]=True`.')
```